csm/conf/configure.py

```diff
@@ -371,16 +369,13 @@ def _create_csm_ldap_user(self):
         ldap_user = Conf.get(const.CSM_GLOBAL_INDEX, const.LDAP_AUTH_CSM_USER)
         csm_ldap_secret =  Conf.get(const.CSM_GLOBAL_INDEX, const.LDAP_AUTH_CSM_SECRET)
         csm_admin_ldap_password = self._fetch_ldap_password(csm_ldap_secret)
-        _output, _err, _rc = self._run_ldap_cmd(f'slappasswd -s {csm_admin_ldap_password}')
-        csm_admin_ldap_password = _output.split('\n')[0]
         ldap_url = Setup._get_ldap_url()
```

Comment:
Please check if 'ldap_url' is required ?

Reply:
Modifications done